server/client_manager.py

```diff
@@ -849,11 +854,15 @@ def change_area(self, area, password=""):
                         )
                     # Something obstructed us.
                     except ClientError:
-                        c.send_ooc(
-                            f"Cannot follow [{self.id}] {self.showname} to [{area.id}] {area.name}!"
-                        )
-                        c.unfollow(silent=True)
-                        raise
+                        if c.forced_to_follow:
+                            c.set_area(area)
+                            f"Following [{self.id}] {self.showname} to [{area.id}] {area.name} by force!."
```

Comment:
This string just exists by itself, it's not being sent to anyone? I think you might've forgotten c.send_ooc(msg)
also, minor punctuation error at the end with !.

Reply:
Instead of ignoring the error and potentially breaking everything, I suggest not doing any special logic for forced_to_follow, and instead just prevent the forced client from /unfollow-ing.